specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/preview/2018-08-20-preview/healthcare-apis.json

```diff
@@ -0,0 +1,982 @@
+{
+  "swagger": "2.0",
+  "info": {
+      "version": "2018-08-20-preview",
+      "title": "HealthcareClient",
+      "description": "Use this API to manage Microsoft Healthcare services in your Azure subscription."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+      "https"
+  ],
+  "consumes": [
+      "application/json"
+  ],
+  "produces": [
+      "application/json"
+  ],
+  "security": [
+      {
+          "azure_auth": [
+              "user_impersonation"
+          ]
+      }
+  ],
+  "securityDefinitions": {
+      "azure_auth": {
+          "type": "oauth2",
+          "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+          "flow": "implicit",
+          "description": "Azure Active Directory OAuth2 Flow",
+          "scopes": {
+              "user_impersonation": "impersonate your user account"
+          }
+      }
+  },
+  "paths": {
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HealthcareApis/services/{resourceName}": {
+          "get": {
+              "tags": [
+                  "Resource"
+              ],
+              "description": "Get the metadata of a service instance.",
+              "operationId": "Service_Get",
```

Comment:
As mentioned over email please ensure operation group name is plural, so this should be Services_Get.

Reply:
Thanks!